src/sql/workbench/browser/ui/infoBox/infoBox.ts

```diff
@@ -98,14 +113,89 @@ export class InfoBox extends Disposable implements IThemable {
 		this.updateStyle();
 	}
 
+	public get links(): azdata.LinkArea[] {
+		return this._links;
+	}
+
+	public set links(v: azdata.LinkArea[]) {
+		this._links = v ?? [];
```

Comment:
You should be refreshing the text as well in case the links changed but the text string didn't